ts/Series/Map/MapPoint.ts

```diff
@@ -220,18 +220,53 @@ class MapPoint extends ScatterSeries.prototype.pointClass {
      * Highmaps only. Zoom in on the point using the global animation.
      *
      * @sample maps/members/point-zoomto/
-     *         Zoom to points from butons
+     *         Zoom to points from buttons
      *
      * @requires modules/map
      *
      * @function Highcharts.Point#zoomTo
      */
     public zoomTo(): void {
-        const point = this as (MapPoint&MapPoint.CacheObject);
-        const chart = point.series.chart;
+        const point = this as (MapPoint&MapPoint.CacheObject),
+            chart = point.series.chart,
+            mapView = chart.mapView;
+
+        let bounds = point.bounds;
+
+        if (mapView && bounds) {
+
+            const inset = isNumber(point.insetIndex) &&
```

Comment:
I would remove this 'enter' / new line after the if statement line. I have checked other HC files and in 99% of cases, there is no new line after the if statement 😊